freetype/__init__.py

```diff
@@ -18,13 +18,22 @@
 import io
 import sys
 from ctypes import *
```

Comment:
No connected to the PR but at some point we should get rid of this `from ctypes import *` in favor of `import ctypes`

Reply:
Agree. There are lots of other code style/PEP8 & related issues that would be good to address (and maybe enforce, via CI) eventually. I didn't want to pollute this PR with such updates though 😄